### ISSUE TITLE
new release into pypi please?

### ISSUE THREAD

**Issue by Artanicus** [hercules-team/python-augeas#42]
The 1.0.3 tag seems to be over a year old and is missing QoL improvements such as:
https://github.com/hercules-team/python-augeas/pull/38/files

**Comment by lutter** [comment]
@Artanicus we seem to have the problem that we do not have a maintainer for python-augeas anymore, and I do not feel well enough versed in the world of Python to act as the maintainer. Would you be interested in taking over maintaining python-augeas, reviewing PR's and issues, and making releases?

**Comment by lutter** [comment]
Maybe I spoke too soon. @thedrow: are you still able to maintain python-augeas? I just noticed that you merged a couple PR's recently. Could you make a new release?

**Comment by thedrow** [comment]
Sure. I'll add it to my TODO list.

**Comment by ptoscano** [comment]
@lutter or maybe we don't have a maintainer for python-augeas, since @thedrow is clearly not doing any releases for almost 2 years, and the latest "stable" release was more than 5 years ago (!).

**Comment by lutter** [comment]
@ptoscano yes, that is really bad. I still don't feel well-equipped to do Python release, and my offer to appoint anybody who raises their hand as maintainer of `python-augeas` still stands. Do you want it?

**Comment by thedrow** [comment]
Releasing now.
Sorry this took so long.

**Comment by thedrow** [comment]
Please try to install 1.1.0 from PyPi.

**Comment by mcanevet** [comment]
@thedrow would it be possible to release a new version? I'd like to use `srun` which is not available in 1.1.0
